jobs/build.Jenkinsfile

```diff
@@ -305,36 +322,44 @@ lock(resource: "build-${params.STREAM}") {
             return
         }
 
-        stage('Kola:QEMU') {
-            // leave 512M for overhead; VMs are 1G each
-            def parallel = ((cosa_memory_request_mb - 512) / 1024) as Integer
+        // reset for the next batch of independent tasks
+        parallelruns = [:]
+
+        // Kola QEMU tests
+        parallelruns['Kola:QEMU'] = {
```

Comment:
We should really look at moving this over to `fcosKola` which already knows to parallelize with upgrade tests.

Reply:
yeah. I'm torn here. I like that all of the actual commands that we know and love are in this file without a level of indirection, but there is a lot of value in the library too.